app/src/androidTest/java/org/mozilla/focus/activity/robots/BrowserRobot.kt

```diff
@@ -318,19 +320,27 @@ class BrowserRobot {
     }
 
     fun longClickAndCopyText(expectedText: String) {
-        var currentTries = 0
-        while (currentTries++ < 3) {
+        for (i in 1..RETRY_COUNT) {
             try {
-                mDevice.findObject(UiSelector().textContains(expectedText)).waitForExists(waitingTime)
-                mDevice.findObject(By.textContains(expectedText)).also { it.longClick() }
+                mDevice.wait(Until.findObject(By.textContains(expectedText)), waitingTime)
+                Assert.assertNotNull(mDevice.findObject(By.textContains(expectedText)))
```

Comment:
could these 2 lines (wait and assertNotNull) be replaced by a single-line waitNotNull? If needed, you can increase the waitingTime to waitingTimeLong.